themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Introducing Azure Native 2.0"
+
+date: 2023-06-13T08:00:00-04:00
+
+meta_desc: "Introducing Pulumi Azure Native Provider 2.0: Streamlined, Expanded, and More Powerful than Ever!"
+
+meta_image: meta.png
+
+authors:
+    - monica-rodriguez
+
+tags:
+    - providers
+    - azure
+
+---
+
+{{% notes type="info" %}}
+The Pulumi Azure Native Provider 2.0 is currently available in beta.
+{{% /notes %}}
+
+We are thrilled to announce the highly anticipated beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to our infrastructure as code (IaC) solution for Microsoft Azure. The native Azure provider implements the best possible support for the Azure platform in Pulumi. Every property of each module is always represented in the SDKs. This release brings a host of exciting features and improvements that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
+
+<!--more-->
+
```

Comment:
I slightly wonder - could we benefit from having a short but "real" code snippet here just to ground this in code using Azure Native?  Not a big deal - I think it also works well without it too.

Reply:
Added a code snippet to the user assigned identities section.